source/01-EQ/07.ptx

```diff
@@ -121,17 +121,37 @@
 </definition>
 
   <activity xml:id="quadratic-inequality-coeff-less-than-0">
-    <statement>
-      <p>Solve the quadratic inequality algebraically<me>2x^2-28 \lt 10x</me>.
-        Write your solution using interval notation.</p>
+    <introduction>
+      <p>Solve the quadratic inequality algebraically<me>2x^2-28 \lt 10x</me>.</p>
+    </introduction>
+    <task>
+      <statement>
+        <p>Write your solution using interval notation.</p>
         <ol marker="A." cols="1">
           <li><p> <m>\left(-\infty,-2\right)\cup\left(7,\infty\right)</m> </p></li>
           <li><p> <m>\left(-\infty,-7\right)\cup\left(2,\infty\right)</m> </p></li>
           <li><p> <m>\left(-7,2\right)</m> </p></li>
           <li><p> <m>\left(-2,7\right)</m></p></li>
-      </ol> 
-    </statement>
-    <answer><p>D.</p></answer>
+        </ol> 
+      </statement>
+      <answer><p>D.</p></answer>
+    </task>
+    <task>
+      <statement>
+      <p>Draw a number line representing your solution.</p>
+      </statement>
+    <answer>
+      <image>
+        <sageplot>
+          <xi:include parse="text" href="../../sage/common.sage"/> 
+          p = TBILPrecal.numberline_plot()
+          p += TBILPrecal.inequality_plot(start=-2,end=7)
+          p.axes(False)
+          p
+        </sageplot>
+      </image>
```

Comment:
@StevenClontz I feel like this ought to work.  If I drop the contents of common.sage into a sage cell, and then these 4 lines it works.

Reply:
I seem to recall that for PreTeXt-CLI to buid PreTeXt, we move all the files to a temporary directory, where we can safely add other necessary files without messing up the file system for the user. But this might have the effect that we're not bringing in `sage/common.sage` to that temporary location with the rest of the source.

I recommend temporarily moving `common.sage` within `source` in this repo. I'll get it all wired up correctly when we port this work to https://github.com/TeamBasedInquiryLearning/library

Reply:
That didn't work either.  It will throw me a "file not found" error if I give it a path to a nonexistant file, which I am not getting.  

The other option is to just do these by hand (there are only 3 of them) and then refactor them later.